client/src-worker/turbo-resolver/registries/npm-http.js

```diff
@@ -13,10 +13,16 @@ export class NpmHttpRegistry {
     }
 
     if (!this.fetching[name]) {
-      this.fetching[name] = fetch(`${this.registryUrl}/${name}`).then(response => {
+      this.fetching[name] = fetch(`${this.registryUrl}/${name}`).then(async response => {
         if(!response.ok){
-          console.error(`Could not load npm registry for ${name}: ${response.statusText}`);
-          throw new Error(response.statusText);
+          // npm can send a json error
+          const dataError = (await (response.json().catch( () => null)));
```

Comment:
Sorry I misread your code.

Reply:
I don't think this new code is justified as all I got from npm registry json error is `{"error":"Not found"}` which tells nothing more than the plain statusText "Not found".

Reply:
Oh, I missed your private repo. Pls show me an json body for private access error.

Reply:
```json
{
  "error": "authorization required to access package aurelia-bootstrapper"
}
```

Reply:
Suggest to simplify the code to:
```js
const dataError = await (response.json().then(j => j.error).catch(() => null));
const error = `Could not load npm registry for ${name}: ${dataError || response.statusText}`;
```